### ISSUE TITLE
Should we use urlClassification to distinguish first/third parties and/or identify tracking?

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/privacy-pioneer#209]
@stanleymarkman pointed out that the `webRequest.onSendHeaders` API has a `urlClassification` object that may be useful for distinguishing first and third parties and/or identify tracking (particularly, fingerprinting). The documentation is [here](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/onSendHeaders) (sort of at the bottom of the page). I do not think that we necessarily need to use it, but if it helps us, we can.

Basically, my understanding is that if a request has been classified by Firefox Tracking Protection as a third party or as performing tracking, we could get that information as well. For example, the documentation explains:

> The classification flags include: fingerprinting and fingerprinting_content: indicates the request is involved in fingerprinting.

@notowen333 (and whoever else wants to chime in), what do you think?
